src/site/content/en/lighthouse-performance/third-party-summary/index.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Reduce the impact of third-party code
+description: |
+  How third-party code like advertising networks and analytics services affect page load performance,
+  and how to optimize third-party code.
```

Comment:
```suggestion
  and how you can optimize third-party code.
```
Thought this might make the syntax a little easier to follow, but not wedded to it.